_docs/tasks/installing-istio.md

```diff
@@ -19,84 +19,75 @@ This page shows how to install and configure Istio in a Kubernetes cluster.
 
   ```bash
   gcloud config set container/use_client_certificate True
+  ```
+
+  Find out your cluster name and zone, and fetch credentials:
+  ```bash
   gcloud container clusters get-credentials <cluster-name> --zone <zone> --project <project-name>
   ```
 
-* Please install [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) or upgrade to the latest version supported by your cluster.
+* Install the Kubernetes client [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/), or upgrade to the latest version supported by your cluster.
 
-* Ensure the `curl` command is present.
+* If you previously installed Istio on this cluster, please uninstall first by following the [uninstalling]({{home}}/docs/tasks/installing-istio.html#uninstalling) steps at the end of this page.
 
-## Installing on an existing cluster
+## Installation steps
 
 For the {{ site.data.istio.version }} release, Istio must be installed in the same Kubernetes namespace as the applications. Instructions below will deploy Istio in the
 default namespace. They can be modified for deployment in a different namespace.
 
-1. Go to [istio release page](https://github.com/istio/istio/releases), and download and extract the installation files `istio.tar.gz`.
+1. Go to the [istio release page](https://github.com/istio/istio/releases), to download the installation file corresponding to your OS.
 
-2. Change directory to install/kubernetes:
+2. Extract the installation file, and change directory to the location where the files were extracted. All instructions on this website are relative to this installation directory.
+   The installation directory contains:
+    * yaml installation files for Kubernetes
+    * sample apps
+    * the `istioctl` client binary, needed to inject Envoy as a sidecar proxy, and useful for creating routing rules and policies.
+    * a text file containing the Istio version.
 
-   ```bash
-   cd install/kubernetes
-   ```
-
-3. Determine if your cluster has [RBAC enabled](https://kubernetes.io/docs/admin/authorization/rbac/) and find out the RBAC api version by running this command:
+3. Run the following command to determine if your cluster has [RBAC (Role-Based Access Control)](https://kubernetes.io/docs/admin/authorization/rbac/)
+enabled:
 
    ```bash
    kubectl api-versions | grep rbac
    ```
    * If the command displays an error, or does not display anything, it means the cluster does not support RBAC, and you can proceed to step 4.
```

Comment:
Applies here and elsewhere.

If you put more than 3 spaces, then there will be extra spaces to the left of the text in the generated HTML. This in turn makes it so when someone selects "Copy" to copy the code block to the clipboard, then those spaces are copied. This becomes a problem when the user then does a "Paste" operation since the spaces are pasted and ruin command-lines.